app/notifications/notifications_ses_callback.py

```diff
@@ -105,7 +110,7 @@ def determine_notification_bounce_type(notification_type, ses_message):
 
 
 def handle_complaint(ses_message):
-    remove_emails_from_complaint(ses_message)
+    recipient_email = remove_emails_from_complaint(ses_message)[0]
```

Comment:
Please confirm this works for a real ses message. You can simulate the response using complaint@simulator.amazonses.com email address.

Reply:
Good call to check it. I simulated the ses complaint response, and checked the logs, and saw the structure of the response after we removed the recipient email, but it did not help me check if the code we wrote would work for it. To do that I needed to see how the recipient email address is stored in the ses response, but our code specifically gets rid of this field before logging it.

Luckily for us, [Amazon handily shares the structure of its ses messages](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/notification-contents.html#mail-object)

Based on that I think the most accurate field to send on to the service team is  `"complainedRecipients"`. I have updated the code accordingly.

Also, from the structure of that field value, quoted below, it looks like there might be more than one recipient who complained about the notification. Hence, I though it would be good to send on the whole value (a list of dictionaries) to the service team. But then maybe it is not how it works and they simply made this structure to account for some edge cases and i should strip it to the email string? What do you think?

```
   "complainedRecipients":[
      {
         "emailAddress":"recipient1@example.com"
      }
   ],
```

Reply:
I just realised we do not have to account for more than one recipient because Notify sends notifications to one recipient per request if I remember correctly?

Reply:
Reverted :)